stories/molecules/featured-highlights-panel.stories.tsx

```diff
@@ -1,19 +1,44 @@
-import { ComponentMeta, ComponentStory } from "@storybook/react";
+import { Meta, StoryFn } from "@storybook/react";
 import FeaturedHighlightsPanel from "components/molecules/FeaturedHighlightsPanel/featured-highlights-panel";
 
 const storyConfig = {
   title: "Design System/Molecules/FeaturedHighlightsPanel",
-} as ComponentMeta<typeof FeaturedHighlightsPanel>;
+} as Meta<typeof FeaturedHighlightsPanel>;
 
 export default storyConfig;
 
-const sampleHighlights = [
-  { id: 1, title: "OAuth, never missing a chance to surprise! 🍕" },
-  { id: 106, title: "Replaced Supabase ui checkbox with Radix UI for better performance" },
-  { id: 106, title: "Even docs can experience breaking changes! 😅" },
+const sampleHighlights: DbHighlight[] = [
+  {
+    id: "1",
+    name: "OAuth, never missing a chance to surprise! 🍕",
```

Comment:
Just a little fun 👍 💯 
```suggestion
    name:'A Pizza Surprise: Unleashing the Power of OAuth! 🍕🔥',
```